packages/core/content-type-builder/admin/src/components/DataManagerProvider/index.js

```diff
@@ -73,9 +72,7 @@ const DataManagerProvider = ({
   const { getPlugin } = useStrapiApp();
 
   const { apis } = getPlugin(pluginId);
-  // FIXME
-  const [, toggleInfoModal] = useState({ cancel: false });
-  // const [infoModals, toggleInfoModal] = useState({ cancel: false });
+  const [infoModals, toggleInfoModal] = useState({ cancel: false });
```

Comment:
I think we can use a boolean here instead of an object since it appears we only use the infoModals.cancel key. Also you might need to change the API when we migrate to the new component.